src/elements/content-explorer/ContentExplorer.js

```diff
@@ -388,16 +388,18 @@ class ContentExplorer extends Component<Props, State> {
     fetchFolderSuccessCallback(collection: Collection, triggerNavigationEvent: boolean): void {
         const { onNavigate, rootFolderId }: Props = this.props;
         const { id, name, boxItem }: Collection = collection;
+        const { selected }: State = this.state;
 
         // New folder state
         const newState = {
+            currentCollection: { ...collection },
```

Comment:
I think this spread is unnecessary because the `collection` passed in as an argument is a new object already from the API layer